sale_packaging_default/models/product_template.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    require_packaging = fields.Boolean(default=False)
```

Comment:
issue: add some help and make sure you cannot require packaging without packaging:

```suggestion
    require_packaging = fields.Boolean(help="Activate it to require that the product is always sold in a complete packaging.")

    @api.constrains("require_packaging", "packaging_ids")
    def _check_packaging_required(self):
        wrong = (self - self.filtered("packaging_ids")).filtered("require_packaging")
        if wrong:
            raise ValidationError(_("Products %s require packaging but have none", ", ".join(wrong.mapped("display_name"))))
```